app/konnect/api-products/productize-service.md

```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
 {% navtabs %}
 {% navtab Konnect UI %}
 
-API products can be published to the Dev Portal. To publish your new API product navigate to the {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and follow these steps: 
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product that you created in the previous step.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**. 
+1. Repeat steps 1-3 for any other Dev Portals you want to add the API product to.
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to. 
```

Comment:
```suggestion
1. Click the product version you created previously and either under the **Actions** button or the **Dev Portals** tab, click **Publish to Dev Portals** and select the Dev Portals you want to add the product version to.

If want to require authentication on your API Product version, toggle **Require Authentication** to true. Note: If you require authentication you will need to select which authentication strategy your API should use. Portal admins can set default auth strategies for their portals so that API Owners don't need to worry about which auth strategy to select. By default, every portal comes with a key auth auth strategy.
```